service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageOrdersAboutToStartControllerTest.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Judge;
+import uk.gov.hmcts.reform.fpl.service.UploadDocumentService;
+import uk.gov.hmcts.reform.fpl.service.docmosis.DocmosisDocumentGeneratorService;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.DocumentMerger;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle.DISTRICT_JUDGE;
+import static uk.gov.hmcts.reform.fpl.enums.State.CLOSED;
+import static uk.gov.hmcts.reform.fpl.enums.State.SUBMITTED;
+
+@WebMvcTest(ManageOrdersController.class)
+@OverrideAutoConfiguration(enabled = true)
+class ManageOrdersAboutToStartControllerTest extends AbstractCallbackTest {
```

Comment:
is ManageOrdersAboutToStartControllerTest  the right name for the test? we are actually testing the mid-event

Reply:
it should be midevent callback test. I renamed it as `ManageOrdersControllerInitialSectionMidEventTest`